README.read_committed.md

```diff
@@ -0,0 +1,40 @@
+## READ COMMITTED transaction isolation
+
+CockroachDB v23.2.0 added support for READ COMMITTED transaction isolation as
+a "preview feature", meaning that we must opt-in to activate it by sending
+the statement
+
+```
+SET CLUSTER SETTING sql.txn.read_committed_isolation.enabled = true;
+```
+
+Unfortunately, SQLAlchemy's "autobegin"
```

Comment:
I think we may not need this warning. The SET CLUSTER SETTING statement only needs to run once, since cluster settings are persisted in storage. They're meant to be run by DBAs to configure the cluster, so it probably wouldn't appear in sqlalchemy application code.

Reply:
Fair enough. The issue will go away when READ COMMITTED graduates from preview feature status anyway.

Feel free to re-word or remove (+ link in CHANGES.md) as desired.

Reply:
i have reworded a bit to highlight what i mentioned above

Reply:
Looks fine to me.